app/views/shelters/show.html.erb

```diff
@@ -7,4 +7,14 @@
   <div class="shelter_options"><%= link_to "Update Shelter", "/shelters/#{@shelter.id}/edit" %></div>
   <div class="shelter_options"><%= link_to "Delete Shelter", "/shelters/#{@shelter.id}", :method => :delete %></div>
   <div class="shelter_options"><%= link_to "See Our Pets", "/shelters/#{@shelter.id}/pets" %></div>
+
+  <section class="review-section">
```

Comment:
Heads up that we've made this class, but we haven't styled it.  Let's make a note on that in the project board.